clients/spark/core/src/main/scala/io/treeverse/clients/Exporter.scala

```diff
@@ -102,7 +104,8 @@ class Exporter(spark : SparkSession, apiClient: ApiClient, filter: KeyFilter, re
 
   private def writeSummaryFile(success: Boolean, commitID: String, content : String) = {
     val suffix = if(success) "SUCCESS" else "FAILURE"
-    val dstPath = resolveURL(new URI(dstRoot), s"EXPORT_${commitID}_${suffix}")
+    val time = ZonedDateTime.now(ZoneOffset.UTC).format(DateTimeFormatter.ISO_INSTANT)
```

Comment:
I don't understand the conversions into a fake timezone.  The requirement has no timezone, so why does the code include one?  [`ISO_INSTANT`](https://docs.oracle.com/javase/8/docs/api/java/time/format/DateTimeFormatter.html#ISO_INSTANT) _is_ "in UTC", per documentation.

```scala
scala> val t = time.Instant.now()
val t: java.time.Instant = 2021-04-07T08:31:10.794768Z

scala> ISO_INSTANT.format(t)
val res7: String = 2021-04-07T08:31:10.794768Z

```

```suggestion
    val t = ISO_INSTANT.format(time.Instant.now()
```

Reply:
Done.